### DISCUSSION TITLE
Welcome to yfinance Discussions!

### DISCUSSION THREAD

**Discussion by ranaroussi** [ranaroussi/yfinance#1068] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FriendlyUser** [comment]
Software developer interested in trading stonks, not going so well this year :(.

Use yfinance every now and then.

**Comment by Victorhpinheiro** [comment]
For now I use yfinance to get signals for interest technical opportunities and then check if the fundamentals make sense. I wanna go deeper and use it for more things but need to have more time.

**Comment by OpsGuyCA** [comment]
Really appreciate all of the effort you are putting in to maintain this useful tool!
